src/main/java/model/Ladder.java

```diff
@@ -0,0 +1,55 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Ladder {
+
+    private static final String NOT_POSITIVE_HEIGHT = "최대 사다리의 높이는 양수가 되어야 합니다";
+    private final int maximumHeight;
```

Comment:
상수와 필드를 분리하면 소소한 가독성을 챙길 수 있습니다 😄
```suggestion
    private static final String NOT_POSITIVE_HEIGHT = "최대 사다리의 높이는 양수가 되어야 합니다";
    
    private final int maximumHeight;
```

Reply:
앗 그렇군요..! 확실히 눈에 더 잘 띄는 것 같습니다. 감사합니다 👍 내용 반영하겠습니다!